### DISCUSSION TITLE
Praca na zewnątrz i zasilanie z 18650

### DISCUSSION THREAD

**Discussion by MiloszAdamek** [hackerspace-silesia/Smogomierz#128] [Ideas]
Witam!

Czy smogomierz byłby w stanie bezpiecznie pracować na zewnątrz? Obudowa ma IP67, chyba wywietrzniki byłby problemem? Gdyby w miejscu samego mUSB zastosować moduł ładowarki ogniw, a za ogniwem przetwornice step-up na 5V to miałoby to rację bytu? Jak wygląda pobór prądu całego układu. Czy można by zoptymalizować kod pod kątem energooszczędności np. pomiar co 15 minut, i tryb czuwania? 

Pozdrawiam

**Comment by bfaliszek** [comment]
Można włączyć w ustawieniach Deepsleep i wybudzanie urządzenia co kilka-kilkanaście minut. 
Jeśli chodzi o obudowę to bez problemu urządzenie może działać na zewnątrz. Sam 5 Smogomierzy, każdy jest w takiej obudowie i jest zasilany przez USB. Najstarszy działa od 3 lat. Wszystkie są na zewnątrz.

**Comment by MiloszAdamek** [comment]
Jak wygląda zasilanie z USB na zewnątrz?

**Comment by bfaliszek** [comment]
No normalnie, mam podłączony kabel do Smogomierza. Chyba nie za bardzo rozumiem o co pytasz.

**Comment by MiloszAdamek** [comment]
Ale jak pociągnięty jest kabel na zewnątrz?

**Comment by bfaliszek** [comment]
najczęściej przekładam go przez okno i zamykam. Da się tak zamknąć okno bez problemu, a kabel nadal działa. Najczęściej tak przerzucam kable USB na zewnątrz.

**Comment by MiloszAdamek** [comment]
A czy obudowa jest wystawiona na działanie np. deszczu?

**Comment by bfaliszek** [comment]
Jasne. Jak już pisałem, przez 3 lata nie miałem problemów z obudową. Tak szczerze to najbardziej problematyczne są kabelki w środki. Czasami trzeba je wymienić po roku, a czasami działają trzy bez problemów

**Comment by MiloszAdamek** [comment]
A z doświadczenia, który czujnik najlepiej zastosować?

**Comment by ookris** [comment]
> Ale jak pociągnięty jest kabel na zewnątrz?

Tak jak pisał Błażej kabel zwyczajnie można "przyciąć" oknem. Polecam do tego kable typu "flat", które bez problemu można tak robić. 

https://www.trust.com/en/product/20135-flat-micro-usb-cable-1m-black

**Comment by bfaliszek** [comment]
Jeśli chodzi o czujnik to polecam Sensirion SPS30 i do tego ESP32. No i BME280 do pomiarów temperatury, wilgotności i ciśnienia.

**Comment by Kriztower** [comment]
![IMG_20200128_193337](https://user-images.githubusercontent.com/60783808/74035645-ee5c6a00-49ba-11ea-8413-1dd6429925d4.jpg)
![IMG_20200128_193326](https://user-images.githubusercontent.com/60783808/74035727-1d72db80-49bb-11ea-9343-92f72304a811.jpg)

U mnie tak to wygląda.

**Comment by jankes46** [comment]
czy jest jakaś możliwość dołożenia panelu słonecznego ?
